GTG/gtk/editor/editor.py

```diff
@@ -204,19 +221,19 @@ def init_accelerators(self):
         agr.connect(key, modifier, Gtk.AccelFlags.VISIBLE, self.new_task)
 
         # Ctrl-Shift-N creates a new subtask
-        insert_subtask = self.builder.get_object("insert_subtask")
+        insert_subtask = self.builder.get_object("add_subtask")
         key, mod = Gtk.accelerator_parse("<Control><Shift>n")
         insert_subtask.add_accelerator('clicked', agr, key, mod,
```

Comment:
Please reuse the references from the above instead. (This means that some can't be removed because they will be used)

```
# Ctrl-Shift-N creates a new subtask
key, mod = Gtk.accelerator_parse("<Control><Shift>n")
self.insert_subtask.add_accelerator('clicked', agr, key, mod,
                                                Gtk.AccelFlags.VISIBLE)
```